modules/installation/ref-rhdh-sizing.adoc

```diff
@@ -0,0 +1,28 @@
+[id='ref-rhdh-sizing_{context}']
+= {product} sizing
```

Comment:
```suggestion
= Sizing requirements for {product}
```
for better clarity

Reply:
Maybe, we can place this section before installing or as part of installing (Chapter2: Installing RHDH). A user needs to know abt this info before they prepare to install the application rather than after installing the application. 
We can build up the context in this manner in chapter 2:
Before installing RHDH, ensure that you have correct component sizing requirements in place for its efficient usage. The following table lists the sizing requirements for installing and running Red Hat Developer Hub, including both the Developer Hub application and Developer Hub database components. 

Administrative users can configure roles, permissions, and other settings to enable other authorized users to install Red Hat Developer Hub on multiple platforms. You can install Developer Hub with a Helm chart or with the Red Hat Developer Hub Operator.

For more information about installing Developer Hub, see..............

Reply:
That's a great point, thanks for adding it. I think we also need a Jira ticket to add sizing as a prerequisite for RHDH installation: [RHIDP-2628](https://issues.redhat.com/browse/RHIDP-2628). For now, I've added this section to Getting Started since we're likely moving the installation sections from the Admin guide to a standalone install title, and the Getting Started guide currently includes an installation section. I've also created a [Jira ticket](https://issues.redhat.com/browse/RHIDP-2629) in the restructure tracker to track the placement of the sizing information.